### ISSUE TITLE
varImpScores.R

### ISSUE THREAD

**Issue by EoghanONeill** [BelindaHernandez/bartBMA#3]
Should varImpScores.R give the importance measure detailed in Hernandez et al. (2018)? If yes, then I think the code should be:

varImpScores<-function(object,...){
  #object will be bartBMA object.
  imp_vars2=get_weighted_var_imp(num_vars=object$numvars,BIC=object$bic,sum_trees=object$sumoftrees)
  res<-apply(imp_vars2[[4]],2,sum)
  #create varImpPlot command
  vIP<-rep(NA,length(res))
  total_weighted_var_counts<-sum(res)
  #now get variable inclusion probabilities
  vIP<-res/total_weighted_var_counts
  class(vIP)<-"varImpScores.bartBMA"
  vIP
}

**Comment by EoghanONeill** [comment]
This is R code, and indexing in R begins at 1, therefore I think the code should be
res<-apply(imp_vars2[[4]],2,sum)
?

**Comment by EoghanONeill** [comment]
I interpreted the names  varImpScores and varIncProb incorrectly. The code that I have suggested for varImpScores and varIncProb swaps around the names of these functions. The code I gave above changes one line of what is currently named varIncProbs, but names the function varImpScores . I also used weighted indicator variables for whether a variable was included in any splits as opposed to counts (in what I called varIncProb, but was originally varImpScores ).
